data/tribes/frisians.lua

```diff
@@ -343,25 +343,60 @@ tribes:new_tribe {
    },
 
    ship_names = {
-      "Amrum",
-      "Föhr",
-      "Gröde",
-      "Habel",
-      "Hamburger Hallig",
-      "Hooge",
-      "Japsand",
-      "Kormoransand",
-      "Langeneß",
-      "Norderoog",
-      "Norderoogsand",
-      "Nordstrand",
-      "Nordstrandischmoor",
-      "Oland",
-      "Pellworm",
-      "Süderoog",
-      "Süderoogsand",
-      "Südfall",
-      "Sylt",
+      -- TRANSLATORS: This frisian ship is named after an island in Northern Frisia
+      pgettext("shipname", "Amrum"),
+      -- TRANSLATORS: This frisian ship is named after a no longer existing island in Northern Frisia
+      pgettext("shipname", "Alt-Nordstrand"),
+      -- TRANSLATORS: This frisian ship is named after a peninsula in Northern Frisia
+      pgettext("shipname", "Eiderstedt"),
+      -- TRANSLATORS: This frisian ship is named after an island in Northern Frisia
+      pgettext("shipname", "Föhr"),
+      -- TRANSLATORS: This frisian ship is named after a Hallig in Northern Frisia
+      pgettext("shipname", "Gröde"),
+      -- TRANSLATORS: This frisian ship is named after a Hallig in Northern Frisia
+      pgettext("shipname", "Habel"),
+      -- TRANSLATORS: This frisian ship is named after a Hallig in Northern Frisia
+      pgettext("shipname", "Hamburger Hallig"),
+      -- TRANSLATORS: This frisian ship is named after an island in the North Sea
+      pgettext("shipname", "Helgoland"),
```

Comment:
This is Heligoland in english

Reply:
"Helgoland" is also used in English: https://dict.leo.org/englisch-deutsch/helgoland